code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -99,11 +99,12 @@ func GetEndpoints(rh rest.RestHandlerI) []rest.Endpoint {
 
 	if config.Development() {
 		restEndpoints = append(restEndpoints, rest.MakeEndpoint(storage+"/all-challenges", srh.getAllChallenges))
+		restEndpoints = append(restEndpoints, rest.MakeEndpoint(storage+"/passed-challenges", srh.getPassedChallengesForBlobberAllocation))
 		restEndpoints = append(restEndpoints, rest.MakeEndpoint(storage+"/block-rewards", srh.getBlockRewards))
 		restEndpoints = append(restEndpoints, rest.MakeEndpoint(storage+"/read-rewards", srh.getReadRewards))
-		restEndpoints = append(restEndpoints, rest.MakeEndpoint(storage+"/challenge-rewards", srh.getChallengeRewards))
 		restEndpoints = append(restEndpoints, rest.MakeEndpoint(storage+"/total-challenge-rewards", srh.getTotalChallengeRewards))
 		restEndpoints = append(restEndpoints, rest.MakeEndpoint(storage+"/cancellation-rewards", srh.getAllocationCancellationReward))
+		restEndpoints = append(restEndpoints, rest.MakeEndpoint(storage+"/acr", srh.getAllocationChallengeRewards))
```

Comment:
let's use the same endpoint format, so could be `/allocation-challenge-rewards` or `/alloc-challenge-rewards` rather that the abbreviation of `/acr`.